aquadoggo/src/replication/service.rs

```diff
@@ -19,21 +19,20 @@ use tokio::task;
 use crate::bus::{ServiceMessage, ServiceSender};
 use crate::context::Context;
 use crate::db::types::StorageEntry;
-use crate::graphql::replication::client;
 use crate::manager::{ServiceReadySender, Shutdown};
 
 /// Replication service polling other nodes frequently to ask them about new entries from a defined
 /// set of authors and log ids.
 pub async fn replication_service(
     context: Context,
     shutdown: Shutdown,
-    tx: ServiceSender,
+    _tx: ServiceSender,
```

Comment:
I'd rather remove everything related to replication for now then having dead code in the codebase. We can always look into the .git history to see how we can reuse code from the past.

Reply:
Sure thing, let's do it.